Adyen/Util/PaymentMethodDetailsConverter.cs

```diff
@@ -140,6 +162,12 @@ public override bool CanConvert(Type objectType)
                 case VisaCheckoutDetails.VisaCheckout:
                     paymentMethodDetails = new VisaCheckoutDetails();
                     break;
+                case WeChatPayDetails.Wechatpay:
+                    paymentMethodDetails = new WeChatPayDetails();
+                    break;
+                case WeChatPayMiniProgramDetails.WechatpayMiniProgram:
+                    paymentMethodDetails = new WeChatPayMiniProgramDetails();
+                    break;
```

Comment:
can you please add `MobilePayDetails` and `UpiDetails` to this converter?

Reply:
I added MobilePayDetails and UpiDetails